ui/src/app/shared/services/tokenizer.ts

```diff
@@ -296,8 +306,8 @@ class Tokenizer {
                     this.lexDoubleQuotedStringLiteral();
                     break;
                 // case '@':
-                // this.pushCharToken(TokenKind.REFERENCE);
-                // break;
+                // 	this.pushCharToken(TokenKind.REFERENCE);
```

Comment:
Lets put these back if/when those are actually used.

Reply:
Agreed. Removed it.